hera_cal/chunker.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2021 the HERA Project
+# Licensed under the MIT License
+
+
+from . import io
+import argparse
+import numpy as np
+import sys
+import warnings
+
+
+def chunk_data_files(filenames, inputfile, outputfile, chunk_size,
+                     polarizations=None, spw_range=None, throw_away_flagged_bls=False,
+                     clobber=False, ant_flag_yaml=None):
+    """A data file chunker
+
+    Parameters
+    ----------
+    filenames: list of strings
+        list of filenames to chunk. Should be homogenous in blt.
+    outpufile: str
+        name of outputfile to write time-concatenated data too.
+    polarizations: list of strs, optional
+        Limit output to polarizations listed.
+        Default None selects all polarizations.
+    spw_range: 2-list or 2-tuple
+        optional lower and upper channel range to select
+    throw_away_flagged_bls: bool, optional
+        if true, throw away baselines that are fully flagged.
+        default is False.
+    clobber: bool, optional
+        if true, overwrite any preexisting output files.
+        defualt is false.
+    flag_yaml : str, optional
+        yaml file with list of antennas to flag and throw away if throw_away_flagged_bls is True
+    Returns
+    -------
+    Concatenated HERAData object.
```

Comment:
The returns section should mirror the `Parameters` section, where the type is on one line and followed by more description. So something like this:
```
Returns
-------
hd : HERAData object
    Concatenated HERAData object.
```
In this case, you have named the return object `hd`, so you can specify that in this docstring to make it easier to find.

Reply:
I've pointed out the name of the object.